### DISCUSSION TITLE
campo pagador no objeto PIX 2.6.3

### DISCUSSION THREAD

**Discussion by faroli** [bacen/pix-api#538] [Q&A] [answered]
Estou com uma dificuldade de encontrar o objeto pagador, verifiquei que na versão **2.0** nas APIs "pix/{e2eid} Consultar Pix" e "/cob/{txid} Consultar cobrança" temos no Schema o objeto PIX com o objeto **pagador** e na versão mais recente como **2.6.3** não encontro mais o objeto pagador, poderiam me explicar o motivo?

**Comment by rubenskuhl** [comment, marked as answer]
Foi removido pelo Banco Central devido a preocupações com privacidade.

**Reply by faroli** [reply]
mas sem esse campo como a instituição que recebe o webhook do pagamento vai mostrar o nome do pagador do PIX no extrato?

**Reply by rubenskuhl** [reply]
> mas sem esse campo como a instituição que recebe o webhook do pagamento vai mostrar o nome do pagador do PIX no extrato?

Esse campo é da API Pix, entre o PSP e o Estabelecimento Comercial. O PSP recebe essa informação do SPI via API 20022, e pode mostrar o pagador dentro do que permite o manual de UX do BACEN (nome citado mas CPF mascarado).

**Reply by faroli** [reply]
desculpe não entendi, entendo que o PSP deve receber esse pagamento via webhook https://bacen.github.io/pix-api/#//post__request_body__webhookUrl__pix, tem outra forma dele receber isso com a informação do pagador?

**Reply by rubenskuhl** [reply]
> desculpe não entendi, entendo que o PSP deve receber esse pagamento via webhook https://bacen.github.io/pix-api/#//post__request_body__webhookUrl__pix, tem outra forma dele receber isso com a informação do pagador?

Não, a API que você está se referindo é apenas para uso dos clientes do PSP, que são os detentores de conta transacional no PSP. Tal como um e-commerce, por exemplo. Os PSPs não usam essa API para se relacionar com o arranjo Pix. Se você é participante, as APIs são outras.

**Reply by faroli** [reply]
entendi, obrigado pela explicação, então para essas APis de participantes não temos acesso a documentação?

**Reply by rubenskuhl** [reply]
> entendi, obrigado pela explicação, então para essas APis de participantes não temos acesso a documentação?

Tem sim. Na data desta publicação, 12.12.2022, a versão mais atual é esta:
https://www.bcb.gov.br/content/estabilidadefinanceira/cedsfn/Manual%20das%20Interfaces%20de%20Comunicacao-1.10.pdf

Mas os endpoints dessas APIs só estão disponíveis dentro da RSFN, que é uma rede segregada... então saber os protocolos não adianta muito.
